src/vivarium_gates_nutrition_optimization/constants/models.py

```diff
@@ -9,6 +9,17 @@ def __new__(cls, value):
         return obj
 
 
+###########################
+# Pregnancy Model         #
+###########################
+PREGNANCY_MODEL_NAME = data_keys.PREGNANCY.name
+# Only one state for now for person-time
+PREGNANCY_STATE = "pregnant"
+PREGNANCY_SUSCEPTIBLE_STATE_NAME = "not_pregnant"
```

Comment:
You may need to subclass the SusceptibleState to make this a reality.

Reply:
Because `SusceptibleState` is subclassed in VPH with `name_prefix` in a basically inaccessible way--
```
class SusceptibleState(BaseDiseaseState):
    def __init__(self, cause, *args, **kwargs):
        super().__init__(cause, *args, name_prefix="susceptible_to_", **kwargs)

I'm having trouble thinking of the cleanest way to do this. You could just subclass `DiseaseState` directly again or modify `SusceptibleState` to take in a `name_prefix` in its init (probably less advisable).

It "feels" like there should be some sort of magic method thing to do this but I don't know what it is. Any Advice?

Reply:
Actually, using `BaseDiseaseState` is problematic because the whole point was to get a `SusceptibleState` into the model in the first place

Reply:
Nvm, I just need to do ‘super(SusceptibleState, self)’ and it should work